PSSwagger.LiveTestFramework/src/PSSwagger.LTF.Lib/Logging/CompositeLogger.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+
+// Licensed under the MIT license.
+namespace PSSwagger.LTF.Lib.Logging
+{
+    using Interfaces;
+    using Newtonsoft.Json;
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.Text;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// Logging interface.
+    /// </summary>
+    public class CompositeLogger : Logger
+    {
+        private IList<Logger> InternalLoggers { get; set; }
+
+        public CompositeLogger() : base(null, null)
+        {
+            this.InternalLoggers = new List<Logger>();
+        }
+
+        public CompositeLogger AddLogger(Logger logger)
+        {
+            this.InternalLoggers.Add(logger);
+            return this;
+        }
+
+        /// <summary>
+        /// Log a single message to output pipe.
+        /// </summary>
+        /// <param name="message">Message to log.</param>
+        public override void Log(string message, params object[] args)
```

Comment:
>message [](start = 40, length = 7)

Consider adding checks on message param here and below functions too.

Reply:
e750658